static_src/components/header.jsx

```diff
@@ -0,0 +1,60 @@
+
+import React from 'react';
+
+import createStyler from '../util/create_styler';
+
+import headerStyle from 'cloudgov-style/css/components/header.css';
+import navStyle from 'cloudgov-style/css/components/nav.css';
+import logoStyle from 'cloudgov-style/css/components/logo.css';
+
+export default class Header extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.styler = createStyler(headerStyle, navStyle, logoStyle);
+  }
+
+  getImagePath(iconName) {
```

Comment:
I'm not sure I understand what you mean by globally. Do you mean in `<App/>` (or something similar) instead of as its own component?